More work needs to be done on the pom.xml to get us to a release.

Follow-up to #7 and #24 

```
[INFO] 
[INFO] --- maven-site-plugin:3.3:deploy (default-deploy) @ aut ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13:53 min
[INFO] Finished at: 2017-08-24T13:37:42-04:00
[INFO] Final Memory: 54M/1667M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:deploy (default-deploy) on project aut: Wagon protocol '' doesn't support directory copying -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

I didn't fully configure the GitHub pages plugin, and we're missing some more release plugins to get us to Sonatype correctly.

--- a/pom.xml
+++ b/pom.xml
@@ -463,7 +463,7 @@
         </plugins>
       </build>
     </profile>
-	</profiles>
+  </profiles>
 
   <pluginRepositories>
     <pluginRepository>

--- a/pom.xml
+++ b/pom.xml
@@ -26,6 +26,13 @@
     <github.global.server>github</github.global.server>
     <checkstyle.plugin.version>2.15</checkstyle.plugin.version>
     <license.plugin.version>2.11</license.plugin.version>
+    <download-maven-plugin.version>1.3.0</download-maven-plugin.version>
+    <release.plugin.version>2.5.2</release.plugin.version>
+    <scm-provider-gitexe.plugin.version>1.9.4</scm-provider-gitexe.plugin.version>
+    <changelog.plugin.version>2.3</changelog.plugin.version>
+    <github-site.plugin.version>0.12</github-site.plugin.version>
+    <javadoc.plugin.version>2.10.3</javadoc.plugin.version>
+    <gpg.plugin.version>1.6</gpg.plugin.version>
   </properties>
 
   <licenses>
@@ -40,6 +47,7 @@
     <connection>scm:git:git@github.com:archivesunleashed/aut.git</connection>
     <developerConnection>scm:git:git@github.com:archivesunleashed/aut.git</developerConnection>
     <url>git@github.com:archivesunleashed/aut.git</url>
+    <tag>HEAD</tag>
   </scm>
 
   <repositories>
@@ -216,8 +224,78 @@
             </goals>
           </execution>
         </executions>
-			</plugin>
-    </plugins>
+      </plugin>
+			<!-- So we can release aut -->
+      <plugin>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>${release.plugin.version}</version>
+        <configuration>
+          <!-- see http://jira.codehaus.org/browse/MRELEASE-424 -->
+          <mavenExecutorId>forked-path</mavenExecutorId>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.maven.scm</groupId>
+            <artifactId>maven-scm-provider-gitexe</artifactId>
+            <version>${scm-provider-gitexe.plugin.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <!-- GitHub Pages -->
+      <plugin>
+        <groupId>com.github.github</groupId>
+        <artifactId>site-maven-plugin</artifactId>
+        <version>${github-site.plugin.version}</version>
+        <configuration>
+          <message>Creating site for ${project.artifactId}, ${project.version}</message>
+          <path>${project.distributionManagement.site.url}</path>
+          <merge>true</merge>
+          <excludes>
+            <exclude>xref-test/**</exclude>
+            <exclude>testapidocs/**</exclude>
+          </excludes>
+        </configuration>
+        <executions>
+          <execution>
+            <id>github</id>
+            <goals>
+              <goal>site</goal>
+            </goals>
+            <phase>site-deploy</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>${javadoc.plugin.version}</version>
+        <configuration>
+          <linksource>true</linksource>
+          <quiet>true</quiet>
+        </configuration>
+        <executions>
+          <execution>
+            <id>verify-javadocs</id>
+            <goals>
+              <goal>jar</goal>
+              <goal>test-jar</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>attach-javadocs</id>
+            <goals>
+              <goal>jar</goal>
+              <goal>javadoc</goal>
+            </goals>
+            <phase>site</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-changelog-plugin</artifactId>
+        <version>${changelog.plugin.version}</version>
+      </plugin>
+		</plugins>
   </build>
 
   <dependencies>
@@ -343,7 +421,7 @@
     <url>https://github.com/archivesunleashed/aut/issues</url>
   </issueManagement>
 
-   <profiles>
+  <profiles>
     <profile>
       <id>release-sign-artifacts</id>
       <activation>
@@ -353,27 +431,39 @@
         </property>
       </activation>
       <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+            </plugin>
+            <plugin>
+              <artifactId>maven-gpg-plugin</artifactId>
+              <version>${gpg.plugin.version}</version>
+              <executions>
+                <execution>
+                  <id>sign-artifacts</id>
+                  <phase>verify</phase>
+                  <goals>
+                    <goal>sign</goal>
+                  </goals>
+                </execution>
+              </executions>
+              <configuration>
+                <useAgent>true</useAgent>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
         <plugins>
           <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
-            <version>1.6</version>
-            <executions>
-              <execution>
-                <id>sign-artifacts</id>
-                <phase>verify</phase>
-                <goals>
-                  <goal>sign</goal>
-                </goals>
-              </execution>
-            </executions>
-            <configuration>
-              <useAgent>true</useAgent>
-            </configuration>
           </plugin>
         </plugins>
       </build>
     </profile>
-  </profiles>
+	</profiles>
 
   <pluginRepositories>
     <pluginRepository>

